Sources/SnowManLogic/View.swift

```diff
@@ -0,0 +1,5 @@
+import OS
```

Comment:
I'd get rid of this. Try and refrain from creating an "production" code without a failing test.